proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -198,9 +198,14 @@ message Service {
   string file_path = 5;
   Location location = 6;
   repeated Method methods = 7;
-  ServiceOptions service_options = 8;
+  ServiceOptions service_options = 8 [deprecated = true];
```

Comment:
Can you move this to the bottom of the message definition, and add a comment "Use options instead."

```
  ...
  repeated FieldLiteral options = 10;
  // Use options instead.
  ServiceOptions service_options = 8;
}
```

Also, is `ServiceOptions` used elsewhere, or is this the only use? If not used elsewhere, also deprecate it:

```
message ServiceOptions {
  option deprecated = true;
  ...
}
```

Apply this across all of the below. Sorry for the back and forth.